### DISCUSSION TITLE
Big Thanks 🙂

### DISCUSSION THREAD

**Discussion by ej52** [themactep/thingino-firmware#248] [Show and tell]
Just wanted to drop a message to say thanks for this amazing project, I have been wanting complete control over my Wyze Cams for so long and this project finally allows to do that 🎉

I can't believe I only found out about this project this week.
I have successfully flashed Wyze Cam v3 and two Wyze Cam Pan v2, and everything is working great.

There are a few issues with webui regarding the saving of Prudynt config and communication with the web socket, but i'll be pushing some PR's up to fix these issues at some point 😉

**Comment by yangminglong** [comment]
> Just wanted to drop a message to say thanks for this amazing project, I have been wanting complete control over my Wyze Cams for so long and this project finally allows to do that 🎉
> 
> I can't believe I only found out about this project this week. I have successfully flashed Wyze Cam v3 and two Wyze Cam Pan v2, and everything is working great.
> 
> There are a few issues with webui regarding the saving of Prudynt config and communication with the web socket, but i'll be pushing some PR's up to fix these issues at some point 😉

People who own Wyze Camera should be interested in knowing about the [wz_flash-helper](https://github.com/gtxaspec/wz_flash-helper) project

**Comment by tavicu** [comment]
I also want to thank you guys. This is incredible work.

Over the last couple of years 4 of my cameras (Wyze v3) just stoped working, couldn't connect to any wifi network ...  I thought that the wifi module burned out due to the power outages (every time they disconnected after a power outage). Every time I tried resetting them, reflashing, and so on but without success. In the end, being that cheap I have replaced them.

Last night I found out about Thingino and I tried on a "burned" camera without much expectations ... to my surprise the camera works perfectly. Wifi is working without problems.

No need to say that I flashed all my cameras to Thingino :P
